app/routes/index.js

```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember';
```

Comment:
This logic should live in a `reminders` route. Generate this with `ember g route reminders` and rethink how we are handing our model to the template. The index route/template should only be used if we need default material on a page, in this issue we want our logic to be associated directly with a `reminders` route. 

Secondly, Mirage is generating 5 reminders for you and popping them into your data store. Once you have the reminders route up created, look into how to find records from the store and pass them to the `reminders.hbs` template.